regulations/views/mixins.py

```diff
@@ -14,6 +14,8 @@ def build_citation(context):
             citation.append(context["part"])
         if 'section' in context:
             citation.append(context["section"])
+        if 'subpart' in context:
+            citation.append(context["subpart"])
```

Comment:
At the moment a `citation` doesn't contain both a section and subpart. Add some logic here to make model that.

Reply:
Resolved in d865569